src/main/java/com/google/api/codegen/transformer/csharp/CSharpGapicSnippetsTransformer.java

```diff
@@ -134,14 +134,49 @@ private SnippetsFileView generateSnippets(GapicInterfaceContext context) {
       } else if (methodConfig.isPageStreaming()) {
         if (methodConfig.isFlattening()) {
           ImmutableList<FlatteningConfig> flatteningGroups = methodConfig.getFlatteningConfigs();
+          // Find flattenings that have ambiguous parameters, and mark them to use named parameters.
+          // These are page-stream flattenings that have one or two string parameters (that are not
+          // resource-names) than another flattening.
+          String[] stringParams =
+              flatteningGroups
+                  .stream()
+                  .map(
+                      flat ->
```

Comment:
Is this "lambda parameter on its own" format a common one in Java? It looks very sparse to me...

Reply:
(Thought I'd already answered this...)

This code is auto-formatted, and the formatting is enforced, so the sparseness cannot be changed,